.sandstorm/Vagrantfile

```diff
@@ -82,7 +82,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     override.vm.synced_folder "..", "/opt/app"
     override.vm.synced_folder ENV["HOME"] + "/.sandstorm", "/host-dot-sandstorm"
-    override.vm.synced_folder "..", "/vagrant"
+    override.vm.synced_folder "..", "/vagrant", disabled: true
```

Comment:
I like to keep VagrantFiles in sync with what is produced by vagrant-spk on first initialization. So maybe vagrant-spk should remove these lines? https://github.com/sandstorm-io/vagrant-spk/blob/aefb8a16c174b71d6fc453db1e11c351f3cd8581/vagrant-spk#L144